### DISCUSSION TITLE
Anyone has a reset password example?

### DISCUSSION THREAD

**Discussion by andresud** [supabase/supabase#1229] [Questions] [answered]
I created a password reset link with redirect like: 
 sendResetPassword(email) {
    return new Promise(async (resolve, reject) => {
      const {
        error,
        data,
      } = await this.supabase.auth.api.resetPasswordForEmail(email, {
          redirectTo: 'http://localhost:8100/new-pw/',
        });
      if (error) {
        reject(error);
      } else {
        resolve(data);
      }
    });
  }

It is generating the email, however when I try to click on the link:
https://jcsjdfmmcekjsjqsmjcd.supabase.co/auth/v1/verify?token=fTpBb15qkorYVX2WHWSuVg&type=recovery&redirect_to=http://localhost:8100/)
to reset the password it is redirecting to: 

http://localhost:8100 instead of http://localhost:8100/new-pw/

if I click the link a second time it redirects to the right place however with this return on the URI:
http://localhost:8100/new-pw#error_code=404&error_description=User+not+found

Any ideas?

**Comment by gomflo** [comment, marked as answer]
You need the change the URL in the `Templates`section look for the `{{ .ConfirmationURL }}` and you can append the path like 

`{{ .ConfirmationURL }}/new-pw` 

![image](https://user-images.githubusercontent.com/141670/115614914-e1702180-a2b3-11eb-9732-5bed7677d949.png)

**Reply by andresud** [reply]
Worked like a charm!!
Thank you! Obrigado! Muchas Gracias!!

**Comment by binajmen** [comment]
Although the answer above clearly works (aka modifying the template), it does not explain why the default `redirectTo` value is not overwritten when a new value is specified in the `options` of the function `resetPasswordForEmail`.

Is it even taken into account or should the doc be updated?
https://supabase.com/docs/reference/javascript/reset-password-email#parameters

No hate, just asking 😄
